manticore/platforms/evm.py

```diff
@@ -1049,6 +1051,41 @@ def __reduce__(self):
         return (self.__class__, (self.data, ))
 
 
+def concretized_args(**policies):
+    '''
```

Comment:
It seems we don't care but maybe triple quotes as [pep257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) suggests

> For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""".